_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,95 @@ nav_order: 50
 Introduced 2.9
 {: .label .label-purple }
 
-Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** interface to:
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly through the **Dashboards** application in OpenSearch Dashboards. 
```

Comment:
Is it the Dashboards application under OpenSearch Dashboards or under Observability?

Reply:
This is the dashboard under OpenSearch Dashboards.